app/recipe/tests/test_recipe_api.py

```diff
@@ -0,0 +1,205 @@
+from django.test import TestCase
+from django.urls import reverse
+from copy import copy
+
+from rest_framework import status
+
+from recipe.models import Recipe, Ingredient
+from recipe.serializers import RecipeSerializer
+from .helpers import sample_recipe, pick
+
+
+def recipe_url(pattern, args=[]):
+    return reverse(f'recipe:recipe-{pattern}', args=args)
+
+
+class RecipeApiTests(TestCase):
+
+    def test_list_recipes(self):
+        """Test retrieving a list of recipes"""
+        sample_recipe()
+        sample_recipe()
+
+        url = recipe_url('list')
+        res = self.client.get(url, {'ordering': 'id'})
+
+        recipes = Recipe.objects.all().order_by('id')
+        serializer = RecipeSerializer(recipes, many=True)
+
+        self.assertEqual(res.status_code, status.HTTP_200_OK)
```

Comment:
If you are using `pytest` you can do as well `assert res.status_code == status.HTTP_200_OK`. I think you will see this more often in our code than using `self.assert*`s, but it is a matter of style.